independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/classloading/QuarkusClassLoader.java

```diff
@@ -706,4 +701,24 @@ public ClassLoader parent() {
             this.parentFirstResources = parentFirstResources;
         }
     }
+
+    /**
+     * Horrible JDK8 hack
```

Comment:
Just passing by, so feel free to ignore me 😉 but I wondered whether this hack could actually be limited to JDK8 only?
Right now it seems to be used generally?

Reply:
I would rather keep it consistent. If we only installed it for JDK8 it would just be one more difference between the environments. getPackage was deprecated in 9 anyway.